### PR TITLE
Implemented more constants

### PR DESCRIPTION
**Pull request by nickdnk**
Hello guys

I've added loads of new constants to different objects, including all possible Stripe error codes to the Base error class, adjusted some minor phpdocs (invoice -> application_fee_amount) and included the constants in tests where applicable.

Let me know if you want me to change or remove anything.

Cheers

### CODE REVIEW

File: lib/Review.php [hunk 1]
```diff
@@ -22,12 +22,25 @@
...
+class Review extends ApiResource
 {
     const OBJECT_NAME = "review";
 
-    use \Stripe\ApiOperations\All;
-    use \Stripe\ApiOperations\Retrieve;
+    use ApiOperations\All;
+    use ApiOperations\Retrieve;
+
+    /**
```
Comment (nickdnk) [inline lib/Review.php]: I put all of the review reasons into a single list of reasons, even though they don't all apply to all of the reason types (open, close and just reason). It didn't make sense to me to create 3 separate reason sections.

File: lib/Review.php [hunk 2]
```diff
@@ -22,12 +22,25 @@
  *
  * @package Stripe
  */
-class Review extends \Stripe\ApiResource
+class Review extends ApiResource
```
Comment (nickdnk) [inline lib/Review.php]: None of the other resources have fully qualified namespaces here, so I removed it.

File: lib/Payout.php [hunk 3]
```diff
@@ -33,6 +33,11 @@ class Payout extends ApiResource
 
     const OBJECT_NAME = "payout";
 
+    use ApiOperations\All;
```
Comment (nickdnk) [inline lib/Payout.php]: Put all of the use statements after OBJECT_NAME in all places where this wasn't the case.

File: lib/Error/Base.php [hunk 4]
```diff
@@ -6,6 +6,98 @@
 
 abstract class Base extends Exception
 {
+
+    /**
+     * All of the possible Stripe error code constants that may be returned from the getStripeCode() method.
+     * @link https://stripe.com/docs/error-codes
+     */
+    const ERROR_ACCOUNT_ALREADY_EXISTS                     = 'account_already_exists';
```
Comment (nickdnk) [inline lib/Error/Base.php]: Dont' worry, I used some clever regex and copy/paste-logic to build this list 😂

File: tests/Stripe/AccountTest.php [hunk 5]
```diff
@@ -254,7 +254,7 @@ public function testCanListPersons()
     public function testSerializeNewAdditionalOwners()
     {
         $obj = Util\Util::convertToStripeObject([
-            'object' => 'account',
+            'object' => Account::OBJECT_NAME,
```
[the remark this answers is not in the document]
Comment (nickdnk) [reply tests/Stripe/AccountTest.php]: Done

File: lib/Error/Base.php (hunk 4 above)
Comment (nickdnk) [reply lib/Error/Base.php]: Made separate PR.

**Comment by nickdnk** [comment]
Changes performed as requested.

**Comment by ob-stripe** [comment]
Released as 6.31.1.

**Pull request merged**